extensions/src/main/scala/pl/touk/nussknacker/extensions/db/pool/DBPoolsConfig.scala

```diff
@@ -0,0 +1,15 @@
+package pl.touk.nussknacker.extensions.db.pool
+
+import com.typesafe.config.Config
+
+object DBPoolsConfig {
```

Comment:
Why it's in another file? Let's do it in `DBPoolConfig` :)

Reply:
This is not a companion object to DBPoolConfig, it's DBPoolSConfig

Reply:
Oh... :) It's misleading.. Maybe it should be named in different way?